packages/bundler-plugin-core/src/utils/providers/AppVeyorCI.ts

```diff
@@ -13,56 +13,87 @@ export function detect(envs: ProviderEnvs): boolean {
   );
 }
 
-function _getBuild(inputs: ProviderUtilInputs): ProviderServiceParams["build"] {
+function _getBuild(
+  inputs: ProviderUtilInputs,
+  output: Output,
+): ProviderServiceParams["build"] {
   const { args, envs } = inputs;
   if (args?.build && args.build !== "") {
+    debug(`Using build: ${args.build}`, { enabled: output.debug });
     return args.build;
   }
-  return envs?.APPVEYOR_JOB_ID ?? null;
+
+  const build = envs?.APPVEYOR_BUILD_ID ?? null;
+  debug(`Using build: ${build}`, { enabled: output.debug });
```

Comment:
Probably not needed right now but devs that are debugging may find it helpful in the future if we log the source as well (as in from `args` or `env`).